### DISCUSSION TITLE
ESP feature on Sigurd's Files

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#360] [Q&A] [answered]
Is it possible to make ESP also shows Sigurd's files?

**Comment by winstxnhdw** [comment]
Sure.

**Reply by Dexteriority** [reply]
Thank you!

**Reply by winstxnhdw** [reply]
Added in latest.

**Reply by Dexteriority** [reply]
I went into Vow, and the ESP doesn't show the log location even though i saw it in front of my eyes :(

**Reply by winstxnhdw** [reply]
Are you on latest? There was an issue earlier.

**Reply by Dexteriority** [reply]
Im sorry, i will try again, i had no problem with the launch.bat since i dont see missing update here:
![image](https://github.com/winstxnhdw/lc-hax/assets/157716300/940f99d1-dbb3-4389-bf3e-60afdd2d99ed)

**Reply by winstxnhdw** [reply]
Alright, let me take a look.

**Reply by Dexteriority** [reply]
the file is named "Data Chip" on scan and its a "Log".
im hoping this information could help what that item actually is

**Reply by winstxnhdw** [reply]
I can only find a class named `StoryLog`, and I am using that right now.

**Reply by Dexteriority** [reply]
It is what makes sense the most.

**Reply by winstxnhdw** [reply]
Okay, I got it. I just messed up translating the coordinates to the screen.

![image](https://github.com/winstxnhdw/lc-hax/assets/56998716/a844a3d2-2947-4b88-844f-1201600cfaba)

**Reply by winstxnhdw** [reply, marked as answer]
Added in latest.

**Reply by Dexteriority** [reply]
Thanks!!
